exampleSite/config.toml

```diff
@@ -91,18 +91,19 @@ paginate = 10
 
     # Since this template is static, the contact form uses www.formspree.io as a
     # proxy. The form makes a POST request to their servers to send the actual
-    # email. Visitors can send up to a 1000 emails each month for free.
+    # email. Visitors can send up to a 50 emails each month for free.
     #
     # What you need to do for the setup?
     #
-    # - set your email address under 'email' below
+    # - register your account to https://formspree.io/register
+    # - login and create new form
+    # - set your form's endpoint under 'formspree_action' below (without https://formspree.io/ from https://formspree.io/sample/of/endpoint )
```

Comment:
remove without, better to have all the URL.

Reply:
Ok. I'll remove 'without' .